resources/definitions/fdmprinter.def.json

```diff
@@ -3290,6 +3290,16 @@
             "description": "Adhesion",
             "children":
             {
+                "prime_enable":
+                {
+                    "label": "Enable Prime",
```

Comment:
I'm not sure whether this naming is good.
Priming is making the printer ready. This involves unretracting as well.

Did you have a talk with somebody about the best name for this setting?

Maybe Menno is the right guy?
Also Jeroen from support might have something to say about this.